src/commands/music/LyricsCommand.ts

```diff
@@ -0,0 +1,85 @@
+import { DefineCommand } from "../../utils/decorators/DefineCommand";
+import { CommandContext } from "../../structures/CommandContext";
+import { BaseCommand } from "../../structures/BaseCommand";
+import { IQueueSong } from "../../typings";
+import { ButtonPagination } from "../../utils/ButtonPagination";
+import { createEmbed } from "../../utils/createEmbed";
+import { AudioPlayerPlayingState } from "@discordjs/voice";
+
+@DefineCommand({
+    aliases: ["ly"],
+    description: "Show the lyrics from current/requested song",
+    name: "lyrics",
+    slash: {
+        description: "Show the lyrics from current/requested song",
+        name: "lyrics",
+        options: [
+            {
+                description: "Song to search",
+                name: "query",
+                type: "STRING",
+                required: false
+            }
+        ]
+    },
+    usage: "{prefix}lyrics [song]"
+})
+export class LyricsCommand extends BaseCommand {
+    public execute(ctx: CommandContext): any {
+        if (ctx.args.length >= 1) {
+            void this.getLyrics(ctx, ctx.args.join(" "));
+        } else {
+            let song;
+            try {
+                song = ((ctx.guild!.queue!.player!.state as AudioPlayerPlayingState).resource.metadata as IQueueSong).song.title;
+            } catch {
+                return ctx.reply({ embeds: [createEmbed("error", "There is nothing playing or no args inserted")] });
+            }
+            void this.getLyrics(ctx, song);
```

Comment:
Maybe you should combine this thing to one? U can do something like this: `ctx.args.join(" ") || ctx.guild!.queue?.player?.state.....` remember to use optional chaining to avoid undefined property access error